packages/components/menu/src/menu-button.ts

```diff
@@ -53,19 +54,9 @@ export class MenuButton extends ScopedElementsMixin(LitElement) {
   /** @internal The menu. */
   @query('sl-menu') menu!: Menu;
 
-  /** Returns the string to be used when there is more than 1 item selected.
-   * Should be used together with `selects = "multiple"`. */
-  @property({ attribute: false }) pluralize?: (count: number) => string;
-
   /** The position of the menu relative to the button. */
   @property() position?: PopoverPosition;
 
-  /** The text representing the selected menuitem(s) that will be shown in the menu button after selection. */
-  @state() selected?: string;
-
-  /** Determines whether if and how many menu items can be selected. */
-  @property() selects?: 'single' | 'multiple';
```

Comment:
Can you please also update the documentation (eg. examples) of the `menu` and change the status there from `planned` to `new`? :)